src/main/resources/org/jenkinsci/plugins/plaincredentials/impl/FileCredentialsImpl/credentials.jelly

```diff
@@ -35,7 +35,7 @@ THE SOFTWARE.
                 <f:textbox field="fileName"/>
             </f:invisibleEntry>
             <f:invisibleEntry>
-                <f:textbox field="data"/>
+                <f:textbox field="secretBytes"/>
```

Comment:
@oleg-nenashev it would cause more confusion to retain the previous field name given that the content of the property has changed.